kafka-rest/src/main/java/io/confluent/kafkarest/controllers/TopicConfigManager.java

```diff
@@ -15,39 +15,37 @@
 
 package io.confluent.kafkarest.controllers;
 
-import io.confluent.kafkarest.entities.TopicConfiguration;
+import io.confluent.kafkarest.entities.TopicConfig;
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.CompletableFuture;
 
 /**
- * A service to manage Kafka {@link TopicConfiguration TopicConfigurations}.
+ * A service to manage Kafka {@link TopicConfig TopicConfigs}.
```

Comment:
I like "A service to manage the configuration of Kafka topics." better and don't think having the `@link` here is hugely necessary (and couldn't think of good words to include it).